ts/parts/Utilities.ts

```diff
@@ -2698,6 +2699,44 @@ const find = H.find = (Array.prototype as any).find ?
         }
     };
 
+/**
+ * Creates a deprecation warning in the console, ones for each context.
+ *
+ * @private
+ *
+ * @param {string} [fnName]
+ * Given function name. This skips stack analysis.
+ *
+ * @param {string} [newFnName]
+ * New function name to use.
+ */
+function deprecate(fnName?: string, newFnName?: string): void {
+    if (!fnName) {
+        try {
+            throw new Error();
+        } catch (err) {
+            if (err instanceof Error) {
+                fnName = (err.stack || '').split('\n')[1];
+            }
+        }
+    }
+    if (
+        fnName &&
+        deprecateFnNames.indexOf(fnName) === -1
+    ) {
+        deprecateFnNames.push(fnName);
+        // eslint-disable-next-line no-console
+        console.log(
```

Comment:
Check if `win.console` is available. Unless we have some polyfill for that, but I guess we don't, since we check that in `H.error()` too. Why? Old IE didn't initiate `win.console` unless dev-tools were open and threw error..